sp800-63b/sec1_2_introduction.md

```diff
@@ -4,29 +4,29 @@
 
 _This section is informative._
 
-This document and its companion documents, [SP 800-63-3](sp800-63-3.html), [SP 800-63A](sp800-63a.html), and [SP 800-63C](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
+This document and its companion documents, [Special Publication (SP) 800-63-3](sp800-63-3.html), [SP 800-63A](sp800-63a.html), and [SP 800-63C](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
```

Comment:
I will add double brackets to links for other SPs - you'll see commits to change sections in -3, 63B, and 63C, and I will include any changes for 63A in the next pull request as well.